src/python/pants/core/util_rules/adhoc_binaries_test.py

```diff
@@ -75,4 +75,4 @@ def test_docker_helper(rule_runner: RuleRunner):
         _PythonBuildStandaloneBinary,
```

Comment:
I note this test is called `test_docker_helper` but uses a `local_environment`; is that expected?